processor/blockprocessor/initialize_test.go

```diff
@@ -63,8 +64,9 @@ func TestRunMigration(t *testing.T) {
 		AlgodToken:     "AAAAA",
 	}
 
+	ctx, cancel := context.WithCancel(context.Background())
 	// migrate 3 rounds
-	err = InitializeLedgerSimple(logrus.New(), 3, &opts)
+	err = InitializeLedgerSimple(ctx, cancel, logrus.New(), 3, &opts)
```

Comment:
Unless you are adding a test that it terminates when you call `cancel()`, these can use `context.Background()`.

Reply:
this method takes a cancel function which is why I used context.WithCancel.